.circleci/config.yml

```diff
@@ -0,0 +1,180 @@
+# CircleCI 2.0 configuration file
```

Comment:
Also check lizard for some improvements regarding testing the build of the docker image.